test/Identity.FunctionalTests/AuthorizationTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Identity.FunctionalTests
+{
+    public class AuthorizationTests
+    {
+        public static TheoryData<string> AuthorizedPages =>
+            new TheoryData<string>
+            {
+                "/Identity/Account/Manage/ChangePassword",
+                "/Identity/Account/Manage/DeletePersonalData",
+                "/Identity/Account/Manage/Disable2fa",
+                "/Identity/Account/Manage/DownloadPersonalData",
+                "/Identity/Account/Manage/EnableAuthenticator",
+                "/Identity/Account/Manage/ExternalLogins",
+                "/Identity/Account/Manage/GenerateRecoveryCodes",
+                "/Identity/Account/Manage/Index",
+                "/Identity/Account/Manage/PersonalData",
+                "/Identity/Account/Manage/ResetAuthenticator",
+                "/Identity/Account/Manage/SetPassword",
+                "/Identity/Account/Manage/ShowRecoveryCodes",
+                "/Identity/Account/Manage/TwoFactorAuthentication",
+                "/Identity/Account/Logout",
+            };
+
+        [Theory]
+        [MemberData(nameof(AuthorizedPages))]
+        public async Task AnonymousUserCantAccessAuthorizedPages(string url)
+        {
+            // Arrange
+            var client = ServerFactory.CreateDefaultClient();
+
+            // Act
+            var response = await client.GetAsync(url);
+
+            // Assert
+            ResponseAssert.IsRedirect(response);
+        }
+
+        // The routes commented below are not directly accessible by
+        // typing the URL in the browser. They have to be accessed as
+        // part of a more complex interation. (like login with 2fa).
+        public static TheoryData<string> RouteableAuthorizedPages =>
+            new TheoryData<string>
+            {
+                "/Identity/Account/Manage/ChangePassword",
+                "/Identity/Account/Manage/DeletePersonalData",
+                //"/Identity/Account/Manage/Disable2fa",
+                "/Identity/Account/Manage/DownloadPersonalData",
+                "/Identity/Account/Manage/EnableAuthenticator",
+                "/Identity/Account/Manage/ExternalLogins",
+                //"/Identity/Account/Manage/GenerateRecoveryCodes",
```

Comment:
commented out

Reply:
It's explained above. These urls are not directly hittable from the browser. The user gets there as part of redirect flows.